README.md

```diff
@@ -91,6 +91,9 @@ using the [Nixpkgs' maintainer list](https://github.com/NixOS/nixpkgs/blob/c51b4
 and its past versions to get the emails,
 falling back to the public GitHub account email.
 
+> [!Warning]
+> The CIVS platform does not accept all valid domains, and will throw an *"Illegal mail address"* error for some. Be sure to check **early** whether your email address is accepted by the platform, and to submit a PR updating `voters.json` if necessary.
```

Comment:
I think this should be in https://github.com/NixOS/SC-election-2024/blob/main/doc/email.md instead, towards the top

Reply:
Didn't even see that file! Updated this.